lib/kino_vega_lite/chart_cell.ex

```diff
@@ -421,7 +421,7 @@ defmodule KinoVegaLite.ChartCell do
           build_arg_field(attrs.x_field,
             type: attrs.x_field_type,
             aggregate: attrs.x_field_aggregate,
-            bin: attrs.x_field_bin,
+            bin: if(attrs.x_field_bin, do: [maxbins: attrs.x_field_bin]),
```

Comment:
Just keep in mind that existing chart cells were saved with this value as either true or false. So you either need to still support booleans or check if the value is an integer.